libs/@guardian/react-crossword/src/components/AnagramHelper.tsx

```diff
@@ -63,6 +63,13 @@ export const AnagramHelper = () => {
 		reset();
 	}, [reset]);
 
+	useEffect(() => {
+		if (showAnagramHelper) {
+			// TODO: Source doesn't support ref forwarding
```

Comment:
I've had this issue before with ref forwarding - do we want to support it in future?

Reply:
Yeah, I don't see why not

Reply:
#261 is already tracking this, so I'll resolve this comment for now